codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -268,7 +332,7 @@ update these file paths accordingly.
 
     // check on results
     Timer.periodic(
-      _notarizationTimerDuration,
+      notarizationTimerDuration,
```

Comment:
For polling, 45 seconds seems like a long interval. What about 5 seconds?

Reply:
Sure, changed to 5 seconds to see if it works out (in case it doesn't i can switch back to 45). I could be wrong but I think previous polls might take more than 3 queries (>135 sec) before the notarization completes.

Reply:
Yes, but that'll take longer for the overall script. If it takes 95 seconds, the script waits an extra 40 seconds. Over 20 artifacts, that adds up to taking an extra 13 minutes.